tox.ini

```diff
@@ -18,7 +18,7 @@ deps =
 
 setenv =
   ; TODO: The two repos branches need manual updated over time, need to figure out a more sustainable solution.
-  CORE_REPO="git+https://github.com/open-telemetry/opentelemetry-python.git@release/v1.22.x-0.43bx"
+  CORE_REPO="git+https://github.com/open-telemetry/opentelemetry-python.git@release/v1.25.x-0.46bx"
   CONTRIB_REPO="git+https://github.com/open-telemetry/opentelemetry-python-contrib.git@main"
```

Comment:
I feel like this should also be `release/v1.22.x-0.43bx` Relying on main seems a bit problematic.

Reply:
I agree. Will change from `main` to release branch for consistency.